### PR TITLE
[ADD] restric access to salesperson and pricelist to salesperson group FGD-Glass

### PR DESCRIPTION
**Pull request by eacm-odoo**
### Description
	-inherit from res.partner:
		-add is_sales_person (computed)
		-create function to compute is_salesperson to know if it belongs to a salesperson group
	-inherit from res_partner_form (form of contacts)
		-show the pricelist
		-make readonly pricelist and salesperson if is_sales_person = true
	-add access rights to the groups

Link to task: [#2874725](https://www.odoo.com/web#id=2874725&cids=17&menu_id=4720&action=4665&active_id=2874696&model=project.task&view_type=form)

### CODE REVIEW

File: odoo_access/security/ir.model.access.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+id,name,model_id/id,group_id/id,perm_read,perm_write,perm_create,perm_unlink
+sale_administrator,sale admin,model_res_partner,sales_team.group_sale_manager,1,1,1,1
+sale_salesperson,sale saleperson,model_res_partner,sales_team.group_sale_salesman_all_leads,1,1,1,1
```
Comment (jefe-odoo) [inline odoo_access/security/ir.model.access.csv:3]: Why did you add this file?

File: odoo_access/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    is_sales_person = fields.Boolean(
+        compute="_compute_is_sales_person", string="Is sales person")
+
+    # function to check if the user is ales_team.group_sale_manager group
+    @api.depends()
```
Comment (jefe-odoo) [inline odoo_access/models/res_partner.py:13]: depends is not always needed

File: odoo_access/views/res_partner_inherit.xml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (jefe-odoo) [inline odoo_access/views/res_partner_inherit.xml:3]: remove data node

**Review by jefe-odoo — CHANGES_REQUESTED** [review]

File: odoo_access/models/res_partner.py (hunk 2 above)
Comment (eacm-odoo) [reply odoo_access/models/res_partner.py:13]: thanks! I'll remeber it

File: odoo_access/security/ir.model.access.csv (hunk 1 above)
Comment (eacm-odoo) [reply odoo_access/security/ir.model.access.csv:3]: my bad forgot to remove it
